pkg/api/login.go

```diff
@@ -45,7 +45,7 @@ func (hs *HTTPServer) validateRedirectTo(redirectTo string) error {
 	return nil
 }
 
-func (hs *HTTPServer) cookieOptionsFromCfg() middleware.CookieOptions {
+func (hs *HTTPServer) CookieOptionsFromCfg() middleware.CookieOptions {
```

Comment:
Do we need the cookies options to be public?

Reply:
Yes, I use it in SAML PostACS() handler to make redirect after login working properly.

Reply:
https://github.com/grafana/grafana-enterprise/blob/a2168a13cf492de8b6a9b2897e501cc68917e5d7/src/pkg/extensions/saml/api.go#L172-L175

Reply:
Ah.